### ISSUE TITLE
Make the license header consistent.

### ISSUE THREAD

**Issue by RaigorJiang** [apache/shardingsphere#21643]
Hi community,

I found that part of the license header format in the project is inconsistent (but the content is consistent)

E.g:
- `.licenserc.yaml`
```yaml
#
# Licensed to the Apache Software Foundation (ASF) under one
# or more contributor license agreements.  See the NOTICE file
# distributed with this work for additional information
...
```

But the license header of **MORE FILES** is like this:
- `.codecov.yml`
```yaml
#
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
...
```

Other files with differences are
- mvnw
- mvnw.cmd
- .gitattributes
- .github/workflows/nightly-graalvm-image.yml
- .mvn/wrapper/maven-wrapper.properties
- src/resources/spotless_shardingsphere_formatter.xml
> Specifically `spotless_shardingsphere_formatter.xml`, Its xml comment format is different from others

I think they should be adjusted to a uniform format.

**Comment by DivvSaxena** [comment]
how can i contribute to this issue?

**Comment by RaigorJiang** [comment]
Hi @DivvSaxena, 
Thanks for your attention. We just need to update the license header of the above file to the same format as the other ones.

**Comment by DivvSaxena** [comment]
Hey @RaigorJiang ,
I think i can do it , assign it to me!

**Comment by RaigorJiang** [comment]
Yes, it should be easy to get started.

**Comment by DivvSaxena** [comment]
Hey @RaigorJiang,   I am not able to find this file .github/workflows/nightly-graalvm-image.yml .

**Comment by linghengqian** [comment]
> Hey @RaigorJiang,   I am not able to find this file .github/workflows/nightly-graalvm-image.yml and 
> Check the PR #21755 

- FYI, this file was deleted some time ago.  Refer to https://github.com/apache/shardingsphere/pull/21677 .

**Comment by DivvSaxena** [comment]
@linghengqian @RaigorJiang As this File is deleted so it's header can't be changed , so all the other files  are adjusted to a uniform format you can merge the PR #21755

**Comment by RaigorJiang** [comment]
@DivvSaxena 
Thanks for the PR, I left a comment for the XML file in it, please check it out

**Comment by DivvSaxena** [comment]
@RaigorJiang  Check it now modified the other part to.

**Comment by RaigorJiang** [comment]
@DivvSaxena Thank you. But I found that there are still differences in the files, can you compare other ones?

**Comment by DivvSaxena** [comment]
@RaigorJiang  Check it now i fixed that problem , I think now it should get merged.

**Comment by RaigorJiang** [comment]
@DivvSaxena CI passed, merged!

**Comment by RaigorJiang** [comment]
Fixed by #21755.

**Comment by DivvSaxena** [comment]
@RaigorJiang Thanks!
